pkg/cstorvolume/v1alpha1/kubernetes.go

```diff
@@ -31,18 +31,25 @@ type getClientsetFn func() (clientset *clientset.Clientset, err error)
 
 // getClientsetFromPathFn is a typed function that
 // abstracts fetching of clientset from kubeConfigPath
-type getClientsetForPathFn func(kubeConfigPath string) (clientset *clientset.Clientset, err error)
+type getClientsetForPathFn func(kubeConfigPath string) (
+	clientset *clientset.Clientset, err error)
```

Comment:
```go
type getClientsetForPathFn func(kubeConfigPath string) (*clientset.Clientset, error)
```

Reply:
The suggested thing was crossing 80 characters. 
Changed it to 
```
type getClientsetForPathFn func(
kubeConfigPath string,
) (*clientset.Clientset, error)
```